modules/remotecontrol/Receiver.js

```diff
@@ -1,11 +1,25 @@
-/* global APP, JitsiMeetJS, interfaceConfig */
-const logger = require("jitsi-meet-logger").getLogger(__filename);
-import {DISCO_REMOTE_CONTROL_FEATURE, REMOTE_CONTROL_EVENT_TYPE, EVENT_TYPES,
-    PERMISSIONS_ACTIONS} from "../../service/remotecontrol/Constants";
-import RemoteControlParticipant from "./RemoteControlParticipant";
+/* global APP, config, interfaceConfig, JitsiMeetJS */
+
 import * as JitsiMeetConferenceEvents from '../../ConferenceEvents';
+import {
+    DISCO_REMOTE_CONTROL_FEATURE,
+    EVENT_TYPES,
+    PERMISSIONS_ACTIONS,
+    REMOTE_CONTROL_EVENT_TYPE
+} from '../../service/remotecontrol/Constants';
+import { getJitsiMeetTransport } from '../transport';
+
+import RemoteControlParticipant from './RemoteControlParticipant';
 
 const ConferenceEvents = JitsiMeetJS.events.conference;
+const logger = require("jitsi-meet-logger").getLogger(__filename);
+
+/**
+ * The transport instance used for communication with external apps.
+ *
+ * @type {Transport}
+ */
+const transport = getJitsiMeetTransport();
```

Comment:
Like the module global `transport` in `API`, `transport` is only necessary once the `constructor` starts executing.